Navigation: Expand & Improve editorType/widget Docs (#8514)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxForm/5 Item Types/SimpleItem/editorType.md

            </ul>
        </td>
        <td>
            <ul>
                <li><a href="/Documentation/Guide/UI_Components/SelectBox/Getting_Started_with_SelectBox/">dxSelectBox</a></li>
                <li><a href="/Documentation/Guide/UI_Components/Slider/Overview/">dxSlider</a></li>
                <li><a href="/Documentation/Guide/UI_Components/Switch/Getting_Started_with_Switch/">dxSwitch</a></li>
                <li><a href="/Documentation/Guide/UI_Components/TagBox/Getting_Started_with_TagBox/">dxTagBox</a></li>
                <li><a href="/Documentation/Guide/UI_Components/TextArea/Getting_Started_with_TextArea/">dxTextArea</a></li>
                <li><a href="/Documentation/Guide/UI_Components/TextBox/Getting_Started_with_TextBox/">dxTextBox</a></li>
            </ul>
        </td>
    </tr>
</table>

[note]

---

##### jQuery

- If you use [DevExtreme modules](/concepts/Common/Modularity/02%20DevExtreme%20Modules%20Structure '/Documentation/Guide/Common/Modularity/DevExtreme_Modules_Structure/'), import the specified editor's module from the `devextreme` folder as follows: 

        import 'devextreme/ui/tag_box';

>     You can omit modules for *"dxTextBox"*, *"dxDateBox"*, *"dxCheckBox"*, and *"dxNumberBox"*. The Form component imports these modules automatically.

Reviewer: ```suggestion
    You can omit the *"dxTextBox"*, *"dxDateBox"*, *"dxCheckBox"*, and *"dxNumberBox"* modules because the Form component imports them automatically.
```

> - Do not specify this property and configure the item component within a [template]({currentpath}/#template) instead to integrate an unsupported component.

Reviewer: ```suggestion
- To integrate an unsupported component, configure the item component in a [template]({currentpath}/#template) instead of specifying this property.
```
